### ISSUE TITLE
Cannot execute polkadot `blocks/3904932`

### ISSUE THREAD

**Issue by emostov** [paritytech/substrate-api-sidecar#445]
**Description**

Cannot decode polkadot block #3904932

**Steps to Reproduce**

api-sidecar version: 3.0.3
ws endpoint: wss://rpc.polkadot.io
query: `/blocks/3904932`

**Expected vs. Actual Behavior**

see https://github.com/polkadot-js/api/issues/3203

**Comment by panlilu** [comment]
3908831/3908816 the same issue

**Comment by mym-xjl** [comment]
node v0.8.28-1 
sidecar v3.0.3

REQ > /blocks/3904932
RET > {
    "code": 500,
    "message": "createType(SignedBlock):: Struct: failed on block: {\"header\":\"Header\",\"extrinsics\":\"Vec<Extrinsic>\"}:: Struct: failed on extrinsics: Vec<Extrinsic>:: createType(ExtrinsicV4):: createType(Call):: Call: failed decoding balances.transfer:: Struct: failed on args: {\"dest\":\"LookupSource\",\"value\":\"Compact<Balance>\"}:: Balance: Input too large. Found input with 164 bits, expected 128",
    "stack": "Error: createType(SignedBlock):: Struct: failed on block: {\"header\":\"Header\",\"extrinsics\":\"Vec<Extrinsic>\"}:: Struct: failed on extrinsics: Vec<Extrinsic>:: createType(ExtrinsicV4):: createType(Call):: Call: failed decoding balances.transfer:: Struct: failed on args: {\"dest\":\"LookupSource\",\"value\":\"Compact<Balance>\"}:: Balance: Input too large. Found input with 164 bits, expected 128\n    at createTypeUnsafe (/hbdata/soft/substrate-api-sidecar/node_modules/@polkadot/types/create/createType.js:72:11)\n    at RpcCore._formatOutput (/hbdata/soft/substrate-api-sidecar/node_modules/@polkadot/rpc-core/index.js:394:41)\n    at callWithRegistry (/hbdata/soft/substrate-api-sidecar/node_modules/@polkadot/rpc-core/index.js:262:42)\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (internal/process/task_queues.js:97:5)",
    "level": "error"
}
same too

**Comment by LmingXie** [comment]
/blocks/3904932  the same issue
```
Error: createType(SignedBlock):: Struct: failed on block: {"header":"Header","extrinsics":"Vec<Extrinsic>"}:: Struct: failed on extrinsics: Vec<Extrinsic>:: createType(ExtrinsicV4):: createType(Call):: Call: failed decoding balances.transfer:: Struct: failed on args: {"dest":"LookupSource","value":"Compact<Balance>"}:: Balance: Input too large. Found input with 164 bits, expected 128
```

**Comment by anhcao142** [comment]
I also have the same issue.

**Comment by dvdplm** [comment]
We have released v3.0.4 that should address this problem. Thank you for the quick reports on this. :)

**Comment by emostov** [comment]
Closed by #446
